### DISCUSSION TITLE
Download on MacOS

### DISCUSSION THREAD

**Discussion by shirlenew** [reach-lab/MixWildGUI#170] [General]
How do I know if I have Rosetta on Apple Silicon

**Comment by rexli999** [comment]
The Apple system will automatically ask you to install Rosetta the first time you install app that needs it. See Apple website https://support.apple.com/en-us/HT211861

**Comment by rexli999** [comment]
Hi Shirlene,

After clearing the configuration, you may continue with the new configuration and click on run model button in order to update the result section.

Jixin
________________________________
